front/src/Phaser/Login/EntryScene.ts

```diff
@@ -24,41 +29,49 @@ export class EntryScene extends Scene {
+                        waScaleManager.applyNewSize();
+                        this.scene.start(nextSceneName);
+                    })
+                    .catch((err) => {
+                        if (err.response && err.response.status == 404) {
+                            ErrorScene.showError(
+                                new WAError(
+                                    "Access link incorrect",
+                                    "Could not find map. Please check your access link.",
+                                    "If you want more information, you may contact administrator or contact us at: hello@workadventu.re"
+                                ),
+                                this.scene
+                            );
+                        } else if (err.response && err.response.status == 403) {
+                            ErrorScene.showError(
+                                new WAError(
+                                    "Connection rejected",
+                                    "You cannot join the World. Try again later" +
+                                        (err.response.data ? ". \n\r \n\r" + `${err.response.data}` : "") +
+                                        ".",
+                                    "If you want more information, you may contact administrator or contact us at: hello@workadventu.re"
+                                ),
+                                this.scene
+                            );
+                        } else {
+                            ErrorScene.showError(err, this.scene);
+                        }
+                    });
             })
-            .catch((err) => {
-                if (err.response && err.response.status == 404) {
-                    ErrorScene.showError(
-                        new WAError(
-                            "Access link incorrect",
-                            "Could not find map. Please check your access link.",
-                            "If you want more information, you may contact administrator or contact us at: hello@workadventu.re"
-                        ),
-                        this.scene
-                    );
-                } else if (err.response && err.response.status == 403) {
-                    ErrorScene.showError(
-                        new WAError(
-                            "Connection rejected",
-                            "You cannot join the World. Try again later" +
-                                (err.response.data ? ". \n\r \n\r" + `${err.response.data}` : "") +
-                                ".",
-                            "If you want more information, you may contact administrator or contact us at: hello@workadventu.re"
-                        ),
-                        this.scene
-                    );
-                } else {
-                    ErrorScene.showError(err, this.scene);
-                }
+            .catch(() => {
```

Comment:
We should really log the error, and rethrow it afterwards to avoid losing it.

```
.catch((e: unknown) => {
   console.error('Some message', e);
   throw e;
});
```